main.go

```diff
@@ -159,38 +159,38 @@ func main() {
 
 		_, err := version.NewVersion(config.NDKVersion)
 		if err != nil {
-			failf(fmt.Sprintf("'%s' is not a valid NDK version. This should be the full version number, such as 23.0.7599858. To see all available versions, run 'sdkmanager --list'", config.NDKVersion))
+			failf(fmt.Sprintf("install dependencies: '%s' is not a valid NDK version. This should be the full version number, such as 23.0.7599858. To see all available versions, run 'sdkmanager --list'", config.NDKVersion))
```

Comment:
`install dependencies` is for the Step's dependency installation phase.
Here, these installations are part of the `run` phase.

Reply:
Are you sure? What is the intended use of the `install dependencies` prefix? Things like installing Ruby gems for eg. Cocoapods?

Reply:
Yes, here is an example: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/main.go#L38

Reply:
Fixed, take a look now @godrei